### PR TITLE
Nf 317 update bank details

### PR DESCRIPTION
**Pull request by njmoorewitchcat**

### CODE REVIEW

File: test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala [hunk 1]
```diff
@@ -300,28 +300,38 @@ class CheckYourAnswersViewSpec extends UnitViewSpec with TestData {
...
-        bankDetailsRow must haveSummaryChangeLinkText(
-          s"${messages("site.change")} ${messages("check_answers.payment.bankDetails.accessible")}"
-        )
+      "have payment section" which {
+        val paymentSection = view().getElementById("payment_section")
+
+        "contains payTo" in {
+          val payToRow = paymentSection.getElementsByClass("pay_to_row")
+          payToRow must haveSummaryKey(messages("check_answers.payment.payTo"))
+          payToRow must haveSummaryValue(MessageKey.apply("check_answers.payment.payTo", repayToAnswer.toString))
```
Comment (SteveSugden) [inline test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala:309]: What happened to the `haveSummaryChangeLinkText` and `haveSummaryActionsHref` checks for pay-to?

And can you add them to the "account name" row test below as well please?

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersView.scala.html [hunk 2]
```diff
@@ -346,27 +346,43 @@
...
-                        claim.bankDetails.accountNumber
-                    ).mkString("<br>"))
+                    content = Text(claim.bankDetails.accountName)
                 ),
                 actions = Some(Actions(
                     items = Seq(
                         actionItem(
                             href = makeclaim.routes.CheckYourAnswersController.onChange(CreatePageNames.bankDetails).url,
                             content = HtmlContent(linkContent(messages("site.change"))),
                             visuallyHiddenText = Some(messages("check_answers.payment.bankDetails.accessible"))
```
Comment (SteveSugden) [inline app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersView.scala.html:361]: Have you checked with Dave that this "alt text" is still suitable?   ("Change bank details").  I believe his gDoc has change links (and alt texts) for each field so he might want to consider the alt text for the single change link

File: test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala (hunk 1 above)
Comment (njmoorewitchcat) [reply test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala:309]: Yes - will do.  When I merged I forgot to add them back in - my bad.  WIll also add the additional ones.  Nice spot

File: app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersView.scala.html (hunk 2 above)
Comment (njmoorewitchcat) [reply app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersView.scala.html:361]: I'm waiting for permission to view the gDoc - which I now have.  Will also do a check if I cannot find the answer there.

File: test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala (hunk 1 above)
Comment (njmoorewitchcat) [reply test/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/views/makeclaim/CheckYourAnswersViewSpec.scala:309]: Actually - when I merged I accidentally duplicated the pay_to_row. There is a perfectly good and complete test above for it, removed duplication, added test for account name change link.

**Review by SteveSugden — APPROVED** [review]
Nice one

**Pull request merged**